src/Widgets/UserBox.vala

```diff
@@ -22,54 +22,60 @@ public class Session.Widgets.Userbox : Gtk.ListBoxRow {
     private const string LOGGED_OFF = _("Logged out");
     private const int ICON_SIZE = 48;
 
-    public Act.User? user { public get; private set; }
-    public bool is_guest = false;
+    public Act.User? user { get; set; }
```

Comment:
Is it necessary to have ```user``` settable from outside?
The ```user``` gets passed as ```UserBox``` is created. I would assume, that the user never gets changed from outside.

What about keeping ```public Act.User? user { public get; private set; }```?